core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -7,6 +9,16 @@ use zksync_types::{
 
 use crate::Tokenizable;
 
+/// Loads KZG settings from the file system.
+pub fn load_kzg_settings() -> KzgSettings {
```

Comment:
Also, I don't think that you need to clone the whole object each time. You can return a `&'static KzgSettings` instead -- `once_cell` allows that.

Reply:
function was removed `load_kzg_settings`